### PR TITLE
(#226) Index.xsd: add a reference to schema

### PR DESCRIPTION
**Pull request by ilyakharlamov**
#226 
The schema fix and a unit test for it

### CODE REVIEW

File: src/test/java/org/jpeek/IndexTest.java [hunk 1]
```diff
@@ -40,18 +41,39 @@
...
+                this.xml
             ),
             XhtmlMatchers.hasXPaths("/index/metric")
         );
     }
 
+    @Test
+    public void xmlHasSchema() {
+        MatcherAssert.assertThat(
+            XhtmlMatchers.xhtml(
```
Comment (paulodamaso) [inline src/test/java/org/jpeek/IndexTest.java]: @ilyakharlamov Please add a failure message here

Comment (ilyakharlamov) [reply src/test/java/org/jpeek/IndexTest.java]: added

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:yegor256/jpeek#317` is not assigned, can't get performer

**Comment by 0crat** [comment]
This job is not in scope

**Pull request merged**